app/src/main/java/org/greenstand/android/TreeTracker/fragments/NewTreeFragment.kt

```diff
@@ -103,6 +107,24 @@ class NewTreeFragment :
                     fragmentNewTreeDBH.text.toString())
             }
         }
+
+        fragmentNewTreeGPS.setOnClickListener {
+            lifecycleScope.launch {
+                progressBar.visibility = View.VISIBLE
+                progressBar.setOnClickListener {
+                    // Do nothing. We want to intercept taps on screen while loading is shown
+                }
+                vm.waitForConvergence()
+                progressBar.visibility = View.GONE
+                fragmentNewTreeSave.isEnabled = true
+                progressBar.setOnClickListener(null)
+            }
+        }
+    }
+
+    override fun onStop() {
+        vm.stopTreeCapture()
```

Comment:
When I checked out your branch for the review, I see this method `stopTreeCapture` is missing and results in compilationError. Can you check?

Reply:
Fixed